### ISSUE TITLE
incorporate some post-processing and/or visualization tools?

### ISSUE THREAD

**Issue by megbedell** [megbedell/wobble#63]
Currently the user is left with a wobble.Results object without much guidance on what to do with it. Some useful things might include:
- [ ] built-in BERV correction convenience function (note: core wobble delivers non-BERV-corrected RVs because that is the observable, therefore this function will never be an automatic part of the wobble model optimization.)
- [ ] built-in HARPS drift correction convenience function
- [ ] prompting to apply the above corrections and/or do `combine_orders()`
- [ ] basic plotting functionality: a simple way to get an RV vs. time plot or a spectral data & model plot for a given order/epoch
- [ ] an interactive plotting widget like @timothydmorton mocked up

**Comment by timothydmorton** [comment]
Sign me up!  Any chance you will be coming to KITP this spring?

**Comment by megbedell** [comment]
Yes I'll be there April 15-26! Does that overlap with your visit?

**Comment by timothydmorton** [comment]
Yup, will be there that whole time! :-)
